app/containers/ChangeLocale/ChangeLocaleButton.tsx

```diff
@@ -24,7 +24,13 @@ const ChangeLocaleButton: React.FC<ChangeLocaleButtonProps> = ({
       fontSize="16"
       lineHeight="20"
       color={colors.commHeadElemColor || 'var(--color-gray-dark)'}
-      onClick={() => setOpen(true)}
+      onClick={
+        typeof setOpen !== 'undefined'
+          ? () => {
+              setOpen(true);
+            }
+          : 'undefined'
+      }
```

Comment:
What is this?
Use
onClick={() => setOpen?.(true)}
or
onClick={() => setOpen && setOpen(true)}